doors_dashboards/components/imprintmodal.py

```diff
@@ -1,28 +1,25 @@
-# components/imprint_modal.py
-
-import dash
-from dash import html, dcc, Input, Output, State
+from dash import dcc
 import dash_bootstrap_components as dbc
 import os
 
-from doors_dashboards.components.constant import ASSETS_PATH
+ASSETS_PATH = "../assets"
```

Comment:
Why not from constant file?

Reply:
Because the path in the constant file is relative to the module. In that way, it only makes sense for that module (or other modules that stand to the assets folder in a similar relation by coincidence), but it is not a general solution.